60-steam-vr.rules

```diff
@@ -23,3 +23,5 @@ KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="28de", ATTRS{idProduct
 KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="28de", ATTRS{idProduct}=="2150", MODE="0660", TAG+="uaccess"
 KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="28de", ATTRS{idProduct}=="2300", MODE="0660", TAG+="uaccess"
 KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="28de", ATTRS{idProduct}=="2301", MODE="0660", TAG+="uaccess"
+
+SUBSYSTEM=="tty", ATTRS{idVendor}=="28de", ATTRS{idProduct}=="2102", MODE="0666", TAG+="uaccess"
```

Comment:
This should *either* be 0666 *or* `uaccess`, probably 0660 and `uaccess`.

0666 means literally any uid, including system uids like `nobody` and `systemd-coredump`, can write to the device - probably not what we want for a peripheral device.

0660 and `uaccess` means that any active, logged-in user can write to the device, but other uids (system uids,or human users logged in via ssh or on an inactive virtual console) cannot. This is usually what we'd prefer.

A better way would be for the stock udev rules shipped by systemd to allow reasonable access (for example https://github.com/systemd/systemd/issues/22681), but so far I have not been able to persuade systemd maintainers that allowing `uaccess` upstream would be preferable to having vendors like Valve (and individual users) patch it in downstream. The broader our requested permissions are, the less likely it is that we will be able to upstream them.

The main objection to using `uaccess` is that after a device has been opened by a user in an active foreground session, they can keep that device open even after their session is no longer the active or foreground session. There's no good answer to this at the udev rules level; we can only prevent that by having a system service that mediates access and can revoke it when necessary (for example for tty1..tty6 and the DRI devices, this is done by logind).